src/frontend/cellprofiler/modules/identifytertiaryobjects.py

```diff
@@ -304,6 +317,7 @@ def run(self, workspace):
         child_count_of_secondary, secondary_parents = secondary_objects.relate_children(
```

Comment:
Considering the assumption that object numbers are continuous and each secondary object is expected to produce exactly one tertiary object, we could replace the `.relate_children()` call here with the expected outputs since it will always be `([1, 1, ...., 1], [1, 2, ..., n])` where `n` is the number of objects.